tests/ArcCommander.Tests.NetCore/ArcCommander/InvestigationTests.fs

```diff
@@ -219,44 +219,45 @@ let testInvestigationContacts =
     ]
     |> testSequenced
 
-[<Tests>]
-let testInvestigationShow =
+// currently not valid anymore since Expecto does not trigger NLog's console loggings. Did not find any workarounds
```

Comment:
So there is no way to check whether commands which print something to the console work properly?

Reply:
Maybe checking the log file since the output gets displayed in the console as well as written into the log file. But this is not the directly tested task...